src/components/Input/index.js

```diff
@@ -127,6 +127,9 @@ Input.propTypes = {
     className: PropTypes.string,
     /** An object with custom style applied to the outer element. */
     style: PropTypes.object,
+    /** The variant changes the appearance of the Input. Accepted variants include default,
+     * and shaded. This value defaults to default. */
+    variant: PropTypes.oneOf(['default', 'shaded']),
```

Comment:
we need to add variant prop to src/components/Input/index.d.ts file, in order to bring typescript support